apps/storybook/.gitignore

```diff
@@ -1,38 +1,24 @@
-# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
As for storybook, we deleted any old storybook config and stories. So if you see deletes of stories, that's why. And diffs in config main that we created a new file, which still had the same name.